File: Algoritmos/Aula03/Aula03_Lista_de_Exercícios/pgm04.c
//Elaborar um programa que efetue a apresentação do valor da conversão em real de um valor lido em 
//dólar. O programa deve solicitar o valor da cotação do dólar e também a quantidade de dólares 
//disponível com o usuário, para que seja apresentado o valor em reais
#include <stdio.h>
int main(void)
{
	double real, dolar,cotacao;
	
	printf("Esse programa e uma calcularadora de conversao de real para dolar \nsiga as instrucoes solicitadas corretamente para opera-la c");
	
	printf("insira valor do real disponivel");
	scanf("%lf",&real);
	
	printf("favor insira a cotação do dia, exemplo: 5.00");
	scanf("%lf",&cotacao);
	
	dolar= real/cotacao;
	printf("voce tem %lf",dolar);
}

File: Programação A/Aula09/ProTrabalho01/src/pgm_04/pgm_04.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package pgm_04;

import javax.swing.JOptionPane;

/**
 *
 * @author LucasSilvaGarcia
 */
/*
4)Elaborar um programa que efetue a apresentação do valor da conversão em real (R$) de um valor lido em
dólar (US$). O programa deverá solicitar o valor da cotação do dólar e também a quantidade de dólares
disponíveis com o usuário.
*/
public class pgm_04 {
    public static void main(String[] args) {
        
            float cotacaodolar, qntdolar, qntreais;
            int resp;

            do{
                String s_cotacaodolar = JOptionPane.showInputDialog("Informe a cotação do dólar de hoje");
                cotacaodolar = Float.parseFloat(s_cotacaodolar);

                String s_qntdolar = JOptionPane.showInputDialog("Informe a quantidade de doláres");
                qntdolar = Float.parseFloat(s_qntdolar);

                qntreais = qntdolar * cotacaodolar;

                JOptionPane.showMessageDialog(null, "A quantidade de reais disponíveis é de:"+qntreais);

                String s_resp = JOptionPane.showInputDialog("Você quer continuar?\n1 - SIM\n2 - NÃO");
                resp = Integer.parseInt(s_resp);

                }while(resp==1);
    
    }    
}
